core/table.ts

```diff
@@ -214,6 +225,19 @@ export class Table {
     });
     this.contextablePostOps = [];
 
+    //Check dependencies are not ambiguous
```

Comment:
we have this in session.ts too, right? i think this can be removed in favour of that check, which also happens at compile()-time.

Reply:
No it wasn't there. I've just added it. Thx.